src/app/pages/datasets/components/dataset-capacity-management-card/dataset-capacity-management-card.component.html

```diff
@@ -64,9 +64,11 @@ <h3 mat-card-title>
       <div class="details-item">
         <span class="label">{{ 'User Quotas' | translate }}:</span>
         <ng-container *ngIf="checkQuotas; else noneTemplate">
-          <span class="value" *ngIf="!isLoadingQuotas; else valuePlaceholder">
-            {{ 'Quotas set for {n} users' | translate: { n: userQuotas } }}
-          </span>
+          <div *ngIf="!isLoadingQuotas; else valuePlaceholder">
```

Comment:
Is it intended to have an extra wrapper here and below?

Reply:
👍
It should be removed.
Done